### PR TITLE
[13.0] [MIG] account_reporting_weight: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT22080

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
Please @chienandalu @sergio-teruel review it

File: account_reporting_weight/reports/account_invoice_report.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+from odoo import fields, models
+
+import odoo.addons.decimal_precision as dp
+
+
+class AccountInvoiceReport(models.Model):
+    _inherit = "account.invoice.report"
+
+    weight = fields.Float(digits=dp.get_precision("Stock Weight"))
```
Comment (chienandalu) [inline account_reporting_weight/reports/account_invoice_report.py]:
```suggestion
    weight = fields.Float(digits="Stock Weight")
```

**Review by chienandalu — COMMENTED** [review]
Tried on runbot but I didn't get any result for the weigh column

**Review by sergio-teruel — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
@chienandalu Changes done!

**Review by chienandalu — COMMENTED** [review]
I see the column blank (I ensured that some of the products had weight):

image

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Great! Now it works :)

**Pull request merged**